targets/hasura/glossary.json

```diff
@@ -0,0 +1,2396 @@
+[
```

Comment:
la sources de donnée initiale

Reply:
I don't know if you should keep into the PR (as the migration script and the sql dump)

Reply:
absolutely not ! It was a simple courtesy if you wanted to test it with a set of data 🙏